linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_legacy_plus_entries_shadow/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: fedora,ocp4,ol7,ol8,rhel7,rhel8,rhv4
+
+title: 'Ensure there are no legacy + NIS entries in /etc/shadow'
+
+description: |-
+    The <tt>+</tt> character in <tt>/etc/shadow</tt> file marks a place where
+    entries from a network information service (NIS) should be directly inserted.
+
+rationale: |-
+    Using this method to include entries into <tt>/etc/shadow</tt> is considered legacy
+    and should be avoided. These entries may provide a way for an attacker
+    to gain access to the system.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: 83390-5
+    cce@rhel8: 83391-3
```

Comment:
CEE `83391-3` is not present in the CCE reference file, it is probably already taken by another rule.